src/airflow/airqo_etl_utils/airqo_api.py

```diff
@@ -178,14 +178,23 @@ def get_app_insights(
         end_time: str,
         frequency: str,
         site_id=None,
+        forecast=False,
+        empty=False,
+        all_data=False,
     ) -> list:
-        params = {
-            "startDateTime": start_time,
-            "endDateTime": end_time,
-            "frequency": frequency,
-            "empty": False,
-            "forecast": False,
-        }
+        if all_data:
+            params = {
+                "startDateTime": start_time,
+                "endDateTime": end_time,
+            }
+        else:
+            params = {
+                "startDateTime": start_time,
+                "endDateTime": end_time,
+                "frequency": frequency,
+                "empty": empty,
+                "forecast": forecast,
+            }
```

Comment:
functionality to enable querying all insights

Reply:
API docs https://platform.airqo.net/api/v1/view/docs/index.html
@Baalmart

Reply:
App insights created in the pipeline and stored in the view microservice via the message broker. @Baalmart